beacon_node/store/src/chunked_vector.rs

```diff
@@ -331,6 +332,23 @@ field!(
     |state: &BeaconState<_>, index, _| safe_modulo_index(state.randao_mixes(), index)
 );
 
+field!(
+    HistoricalSummaries,
+    VariableLengthField,
+    HistoricalSummary,
+    T::HistoricalRootsLimit,
+    DBColumn::BeaconHistoricalSummaries,
+    |_| OncePerNSlots {
+        n: T::SlotsPerHistoricalRoot::to_u64()
+    },
+    |state: &BeaconState<_>, index, _| safe_modulo_index(
+        state
+            .historical_summaries()
+            .map_err(|_| ChunkError::InvalidFork)?,
+        index
+    )
+);
+
```

Comment:
Ah I'd forgotten we had the historical roots in the freezer DB, but this looks right for putting historical summaries there too